### ISSUE TITLE
Cannot set device-specific + touchpad settings at same time

### ISSUE THREAD

**Issue by hvitoi** [hyprwm/Hyprland#2895]
### Hyprland Version

46cf4eb837c39255f24fa32b5a740c28ce1a324f

### Bug or Regression?

Bug

### Description

When setting a touchpad device-specific configuration (`device:mytouchpad`), the default input config at `input:touchpad:` is ignored

This behavior differs from the [Per-device input configs documentation](https://wiki.hyprland.org/Configuring/Keywords/#per-device-input-configs) which states that "only values explicitly changed will be overwritten."

### How to reproduce

Given the configuration:

```ini
input {
    touchpad {
        natural_scroll = true
        tap-to-click = false
    }
}

device:bcm5974 {
    sensitivity = 0.75
}
```

## Expected Behavior

- Mouse is kept the default sensitivity
- Mouse is kept conventional scroll direction
- Touchpad (bcm5974) has an increased sensitivity
- Touchpad (bcm5974) has natural scroll direction
- Touchpad (bcm5974) has tap to click disabled

## Observed Behavior

- Mouse is kept the default sensitivity (OK)
- Mouse is kept conventional scroll direction (OK)
- Touchpad (bcm5974) has an increased sensitivity (OK)
- Touchpad (bcm5974) has **conventional** scroll direction (NOK)
- Touchpad (bcm5974) has tap to click **enabled** (NOK)

### Crash reports, logs, images, videos

```shell
➜  ~ sudo libinput list-devices

...

Device:           bcm5974
Kernel:           /dev/input/event9
Group:            7
Seat:             seat0, default
Size:             104x75mm
Capabilities:     pointer gesture
Tap-to-click:     disabled
Tap-and-drag:     enabled
Tap drag lock:    disabled
Left-handed:      disabled
Nat.scrolling:    disabled
Middle emulation: disabled
Calibration:      n/a
Scroll methods:   *two-finger edge 
Click methods:    button-areas *clickfinger 
Disable-w-typing: enabled
Disable-w-trackpointing: enabled
Accel profiles:   flat *adaptive custom
Rotation:         n/a

Device:           MX Anywhere 3 Mouse
Kernel:           /dev/input/event18
Group:            9
Seat:             seat0, default
Capabilities:     pointer 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      disabled
Nat.scrolling:    disabled
Middle emulation: disabled
Calibration:      n/a
Scroll methods:   button
Click methods:    none
Disable-w-typing: n/a
Disable-w-trackpointing: n/a
Accel profiles:   flat *adaptive custom
Rotation:         0.0
```

```shell
➜  ~ hyprctl devices

...

mice:
	Mouse at 55c727a353a0:
		bcm5974
			default speed: 0.000000
	Mouse at 55c727f07590:
		mx-anywhere-3-mouse
			default speed: 0.000000

Touch:


```

**Comment by MightyPlaza** [comment]
looks like https://github.com/hyprwm/Hyprland/issues/2483
idk if it got broken again or never got fixed

**Comment by vaxerski** [comment]
maybe broken again

**Comment by pdamianik** [comment]
Should get fixed with #3184
